library/L2_HAL/sensors/environment/light/temt6000x01.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "L2_HAL/sensors/environment/light.hpp"
+
+#include "L1_Peripheral/adc.hpp"
+#include "utility/log.hpp"
+
+namespace sjsu
+{
+class Temt6000x01 final : public Light
+{
+ public:
+  /// Pull-down resistor
+  static constexpr uint32_t kResistance = 10'000;
+  /// 12-bit ADC resoltion = 2^12 - 1 = 4'095
+  static constexpr uint32_t kResolution = 4'095;
```

Comment:
This is platform dependent. Use Adc::GetActiveBits() to get the resolution of the passed adc object.